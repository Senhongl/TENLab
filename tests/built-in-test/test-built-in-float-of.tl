a = [1,2,3];
c = float_of(a);
print(c);

b = [2.1,3.4,6.7];
print(c * b);