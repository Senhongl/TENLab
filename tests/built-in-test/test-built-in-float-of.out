 1
 2
 3
[ CPUDoubleType{3} ]
29
[ CPUDoubleType{} ]
